sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -1513,6 +1584,8 @@ def repeat(self, repeats):
               'repeat',
               lambda series: series.str.repeat(repeats),
               [self._expr],
+              # Output will also be a str Series
```

Comment:
Is there a drawback to this being automatically inferred? (Or was it not?)

Same below.

Reply:
Yeah when I added the verification of the proxy in `TransformTest` I found that this proxy was incorrectly inferred as `bool` for the zipping case tested there.

```py
In [10]: proxy.dtypes
Out[10]: 
str        object
repeats     int64
dtype: object

In [11]: proxy.str.str.repeat(proxy.repeats)
Out[11]: Series([], Name: str, dtype: bool)
```

The actual operation does produce `object` though:
```py
In [13]: df.str.str.repeat(df.repeats)
Out[13]: 
0      AAA
1        B
2     CCCC
3    DDDDD
4       EE
Name: str, dtype: object
```

Reply:
Maybe something we should fix upstream and patch around here in the meantime?

Reply:
Hmm... OK, can you file a JIRA to investigate?